src/test/java/io/vertx/ext/mongo/impl/UtilsTest.java

```diff
@@ -0,0 +1,87 @@
+package io.vertx.ext.mongo.impl;
+
+import com.mongodb.bulk.BulkWriteInsert;
+import com.mongodb.bulk.BulkWriteResult;
+import com.mongodb.bulk.BulkWriteUpsert;
+import io.vertx.ext.mongo.MongoClientBulkWriteResult;
+import io.vertx.test.core.TestUtils;
+import org.bson.BsonString;
+import org.junit.Test;
+import java.util.Collections;
+import java.util.List;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNull;
+
+/**
+ * @author <a href="mailto:nikolakofistariji@gmail.com">Nikola Kosanovic</a>
+ */
+public class UtilsTest
+{
+  @Test
+  public void noAcknowledgedTest() {
```

Comment:
Given the `Utils` class does not only handle `MongoClientBulkWriteResult`, the test name should indicate which type of data transformation we're testing, I think. Like `testUnacknowledgedBulkWriteResult()`